doc/rtd/topics/instancedata.rst

```diff
@@ -126,6 +150,14 @@ Examples output:
 
 - i-<hash>
 
+v1.kernel
+---------
+This shall the running kernel `uname -r`
+
+Example output:
+
+- 5.3.0-1010-aws
```

Comment:
I wonder if we want to change the key name here;  this is the kernel_release value.
This allows us later on add other keys around systematic parsing of the string, like the major, minor, micro values.

Reply:
renamed kernel_release